### DISCUSSION TITLE
Bienvenidos 🤗

### DISCUSSION THREAD

**Discussion by Noelia-Ruiz** [CodeSystem2022/Perricornios-Tercer-Semestre#197] [Team Posts]
Sean bienvenidos chicos, estamos dentro de lo que es tecnicatura asi que comportensé 🤣

**Comment by ArielBetancud22** [comment]
**Estoy esperando que ingresen todos, me avisas Noelia, por cierto: excelente trabajo**

**Comment by Noelia-Ruiz** [comment]
Gracias profe, nos faltan solo dos y estamos 👌

**Comment by NaSP32** [comment]
> Sean bienvenidos chicos, estamos dentro de lo que es tecnicatura asi que comportensé

Muchas gracias Noe por la bienvenida :) 
Aqui dejamos (de paso), el registro de la asistencia de hoy 30/9

**Comment by Noe3467** [comment]
Ahora si profe, equipo Perricornios completo! 👏

**Comment by Noe3467** [comment]
Ya está completa la planilla de excel también ✍️

**Comment by asterisko19** [comment]
ahora si a portarse mal jeje

**Comment by ArielBetancud22** [comment]
**Genial, grupo completo, saludos a todos**
